src/saving_files.jl

```diff
@@ -24,6 +24,8 @@ end
 ```
 
 ## Keywords
+* `filename = savename(prefix, config, suffix; kwargs...)` : Name of the file
+  to produce or load, relative to `path`. The output `filename` includes `path`.
```

Comment:
It is not necessary to say "The output `filename` includes `path`.", it is already stated in the previous part of the docstring. However, it would be useful to mention the reason for this keyword, i.e., your use case. Something like:

"This may be useful in situations where `config` has too many parameters for `savename` to be useful, and an explicitly specified name is more suitable."

Reply:
Sure, sounds good! I'll commit that.

What workflow do you prefer? Add a third commit with fixes based on the review, or amend my previous commits and force-push? Do you want me to squash the actual patch and the version/changelog update into a single commit?

Reply:
It does not matter, feel free to use whatever workflow suits you best. No matter what, I will squash-merge the pull request anyways, so all will be pushed into master as 1 commit.